### PR TITLE
[BugFix] Fix multiply result error

### LINKED ISSUE #15455: [Bug] Unexpected Behavior for multipy function
### Steps to reproduce the behavior (Required)
image

### StarRocks version (Required)
main

### PR DESCRIPTION
**Pull request by Seaven**
## What type of PR is this：
- [x] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes https://github.com/StarRocks/starrocks/issues/15455

multiply/add/subtract/divid use function call expr analyze, not arithmetic expr, return type is different

## Checklist:

- [x] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3
  - [ ] 2.2

### CODE REVIEW

**Comment by Seaven** [comment]
run all

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 60 / 63 (95.24%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/analyzer/ExpressionAnalyzer.java|35|38|92.11%|[519, 685, 686]|
|:large_blue_circle:|com/starrocks/analysis/ArithmeticExpr.java|18|18|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/optimizer/rewrite/ScalarOperatorFunctions.java|7|7|100.00%|[]|

**Review by kangkaisen — DISMISSED** [review]

File: fe/fe-core/src/main/java/com/starrocks/analysis/ArithmeticExpr.java [hunk 1]
```diff
@@ -47,11 +47,30 @@
 import com.starrocks.thrift.TExprNode;
 import com.starrocks.thrift.TExprNodeType;
 import com.starrocks.thrift.TExprOpcode;
+import org.spark_project.guava.collect.ImmutableMap;
```
Comment (HangyuanLiu) [inline fe/fe-core/src/main/java/com/starrocks/analysis/ArithmeticExpr.java:50]: import error

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 63 / 66 (95.45%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/analyzer/ExpressionAnalyzer.java|35|38|92.11%|[519, 685, 686]|
|:large_blue_circle:|com/starrocks/analysis/ArithmeticExpr.java|21|21|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/optimizer/rewrite/ScalarOperatorFunctions.java|7|7|100.00%|[]|

**Comment by Seaven** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 60 / 66 (90.91%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|4|7|57.14%|[4530, 4531, 4532]|
|:large_blue_circle:|com/starrocks/sql/analyzer/ExpressionAnalyzer.java|28|31|90.32%|[519, 686, 687]|
|:large_blue_circle:|com/starrocks/analysis/ArithmeticExpr.java|21|21|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/optimizer/rewrite/ScalarOperatorFunctions.java|7|7|100.00%|[]|

**Review by HangyuanLiu — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 61 / 65 (93.85%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|5|6|83.33%|[4561]|
|:large_blue_circle:|com/starrocks/sql/analyzer/ExpressionAnalyzer.java|28|31|90.32%|[519, 686, 687]|
|:large_blue_circle:|com/starrocks/analysis/ArithmeticExpr.java|21|21|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/optimizer/rewrite/ScalarOperatorFunctions.java|7|7|100.00%|[]|

**Review by HangyuanLiu — APPROVED** [review]

**Review by kangkaisen — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Pull request merged**